src/webgl-utils/track-context-state.js

```diff
@@ -45,9 +45,10 @@ export const GL_STATE_SETTERS = {
 
   // SPECIFIC SETTERS
 
-  clearStencil: (setter, s) => setter({
-    [GL.STENCIL_CLEAR_VALUE]: s}
-  ),
+  bindFramebuffer: (setter, target, fb) => setter({
```

Comment:
Per my read of the article I forwarded, it should be 
```
bindFramebuffer: (setter, target, fb) => {
   switch (target) {
   case GL.FRAMEBUFFER:  
      setter({
         [GL.READ_FRAMEBUFFER_BINDING]: fb,
         [GL.DRAW_FRAMEBUFFER_BINDING]: fb
      });
     break;
   case GL.DRAW_FRAMEBUFFER:  
      setter({[GL.DRAW_FRAMEBUFFER_BINDING]: fb});
     break;
  case GL.READ_FRAMEBUFFER:
    setter({[GL.DRAW_FRAMEBUFFER_BINDING]: fb});
    break;
  }
})
```

Reply:
I moved this logic into `context-state.js`, which actually handles function style settings.